indra/llcommon/is_approx_equal_fraction.h

```diff
@@ -45,7 +45,7 @@
 template <typename FTYPE>
 inline BOOL is_approx_equal_fraction_impl(FTYPE x, FTYPE y, U32 frac_bits)
```

Comment:
You forgot the return type and the type for "ret" two lines below.

Reply:
Where is that file included from? It is nowhere to be found in the Xcode project....